### PR TITLE
New line character added at the end of text files

### PR DESCRIPTION
**Pull request by mtochi**
New line character added at the end of all text files

### CODE REVIEW

**Comment by mtochi** [comment]
@SergioRAgostinho, could you please explain to me what should I do? I don't understand why it hasn't passed the checks. What could be a possible problem?

**Comment by mtochi** [comment]
So, all I need to do is at the end of the row (in the first pic) click 'enter', and that's it?
And change the others that have extra new lines.

**Comment by SergioRAgostinho** [comment]
> So, all I need to do is at the end of the row (in the first pic) click 'enter' and delete the '\n', and that's it?

Yes, although in my first screenshot there's no `\n` so you would not need to delete it.  It might be easier to start from scratch.

**Comment by mtochi** [comment]
So I should keep the \n but just delete the other extra lines?

**Comment by SergioRAgostinho** [comment]
> So I should keep the \n but just delete the other extra lines?

No. You should delete all \n you added.

**Comment by mtochi** [comment]
I believe now it's solved.

File: gpu/people/data/results/README [hunk 1]
```diff
@@ -1 +1,2 @@
 Please download the trees from https://github.com/PointCloudLibrary/data/tree/master/people/results into this directory.
+\n
```
Comment (kunaltyagi) [inline gpu/people/data/results/README]: This appears to have slipped by

File: test/rops_indices.txt [hunk 2]
```diff
@@ -997,4 +997,4 @@
 29433
 5088
 1713
-5038
\ No newline at end of file
+5038
```
Comment (kunaltyagi) [inline test/rops_indices.txt]: We can't see CRLF or CR line endings on Github. Can someone please verify that this is the correct diff? (I can't do it, sorry)

File: test/rops_triangles.txt [hunk 3]
```diff
@@ -57616,4 +57616,4 @@
 32087 31453 32086
 31454 30824 31453
 30823 30824 30504
-30823 31453 30824
\ No newline at end of file
+30823 31453 30824
```
Comment (kunaltyagi) [inline test/rops_triangles.txt]: Same as `rops_triangles`

File: test/rops_indices.txt (hunk 2 above)
Comment (SergioRAgostinho) [reply test/rops_indices.txt]: I was actually going to propose to not touch data files like these.

Comment (kunaltyagi) [reply test/rops_indices.txt]: Time to test the tests 🤣

Comment (SergioRAgostinho) [reply test/rops_indices.txt]: I would say so. This is not our code. It's data.

Comment (kunaltyagi) [reply test/rops_indices.txt:1000]: @mtochi please undo the changes in these two text files

Comment (mtochi) [reply test/rops_indices.txt]: @kunaltyagi I did that, but it's still failing two checks, what could be the problem?

**Comment by mtochi** [comment]
I don't understand why it failed the two checks?

**Comment by SergioRAgostinho** [comment]
It has nothing to do with your changes. Don't worry about it.

**Pull request merged**